packages/eslint-plugin-template/tests/rules/accessibility-table-scope.test.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  convertAnnotatedSourceToFailureCase,
+  RuleTester,
+} from '@angular-eslint/utils';
+import rule, {
+  MessageIds,
+  RULE_NAME,
+} from '../../src/rules/accessibility-table-scope';
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parser: '@angular-eslint/template-parser',
+});
+
+const messageId: MessageIds = 'accessibilityTableScope';
+
+ruleTester.run(RULE_NAME, rule, {
+  valid: ['<th scope="col"></th>', '<th [attr.scope]="col"></th>'],
+  invalid: [
+    convertAnnotatedSourceToFailureCase({
+      messageId,
+      description: 'should fail when element other than th has scope',
+      annotatedSource: `
+        <div scope></div>
+             ~~~~~
+      `,
+    }),
+    convertAnnotatedSourceToFailureCase({
+      messageId,
+      description: 'should fail when element other than th has scope input',
+      annotatedSource: `
+        <div [attr.scope]="scope"></div>
```

Comment:
@JamesHenry the `BoundAttribute` visitor is not called for `attr.` bindings. Am I missing anything in the code?

Reply:
You’ll probably just need to update the visitor keys in the parser again right?